### PR TITLE
Fix Shortcut Ambiguities, Clean up text

### LINKED ISSUE #126: Shortcut ambiguities
An example of shortcut ambiguity:
DeepinScreenshot_select-area_20201031160227

Pressing `Alt+D` a user could expect switching to the "Display" tab, but actually the "Size of database cache" widget is activated. The following `Alt+D` routes to the "Display" tab.

Other tabs of the "Options" window also have such ambiguities.

Localized versions of the GUI have such ambiguities too.

### PR DESCRIPTION
**Pull request by jarolrod**
Removes shortcut ambiguities found in the Settings -> Options menu. Builds upon this [rationale](https://github.com/bitcoin-core/gui/issues/126#issuecomment-720837550).

#### Before/Master
master

#### After/PR
pr

#### Main Tab
1. `Start Bitcoin Core on system login`
   - Moved shortcut from [**S**]tart to -> [**B**]itcoin in order to reserve S for `Size of database cache`
2. `Prune block storage to`
   - Moved shortcut from [**B**]lock to -> [**P**]rune in order to reserve B for `Start Bitcoin Core on system login`
   - Added a colon at the end since this text denotes a data field
3. `Size of database cache`
   - Moved shortcut from [**D**]atabase to -> [**S**]ize in order to fix the conflict with the `Display` tab shortcut
   - Added a colon at the end since this text denotes a data field
4. `Number of script verfication threads`
   - Added a colon at the end since this text denotes a data field

#### Wallet Tab
1. `Enable coin control features`
   - Moved shortcut from [**C**]oin to -> [**E**]nable in order to fix conflict with the `Cancel` button

#### Network Tab
1. `Connect through SOCKS5 proxy (default proxy):
   - Moved shortcut from [**C**]onnect to -> SOCKS[**5**] in order to fix conlict with the `Cancel` button

#### Window Tab
1. `Minimize to the tray instead of the taskbar`
   - Moved shortcut from [**M**]inimize to -> [**T**]ray in order to fix conflict with the `Main` tab

Closes #126

### CODE REVIEW

**Comment by hebasto** [comment]
Concept ACK.

The "GUI:" prefix is redundant in the GUI repo PR title (but the "qt: " prefix in the commit message is correct).

Generally, screenshots "before"/"master" and "after"/"pr" are expected as a part of the OP in the GUI repo :)

**Review by hebasto — COMMENTED** [review]
Approach ACK 401a5e70de42e6ddf0b5bc79d96ea8d670851860.

About adding `:` character:
- "Third party transaction URLs" string could be ended with `:`
- "Prune block storage to <NN> GB" seems a complete simple sentence, and adding an `:` is not required, but I'm not an expert in the field of English punctuation rules :)

**Comment by hebasto** [comment]
ping @Bosch-0 @GBKS @johnsBeharry

**Comment by Bosch-0** [comment]
Testing ACK 7e373294a5ae819099c39d9d03d1f5a311d63cfc on Windows Windows 10.0.18363 Build 18363 and Ubuntu 20.04.

Looks good to me!

**Comment by jarolrod** [comment]
> Approach ACK [401a5e7](https://github.com/bitcoin-core/gui/commit/401a5e70de42e6ddf0b5bc79d96ea8d670851860).
> 
> About adding `:` character:
> 
>     * "Third party transaction URLs" string could be ended with `:`
> 
>     * "Prune block storage to <NN> GB" seems a complete simple sentence, and adding an `:` is not required, but I'm not an expert in the field of English punctuation rules :)

While that is a complete sentence, I think the "Prune block storage to: [NN] GB" makes this field visually and functionally consistent with the other data fields it is surrounded by.

**Comment by hebasto** [comment]
> About adding `:` character:
> 
>  - "Third party transaction URLs" string could be ended with `:`

?

**Comment by hebasto** [comment]
@Bosch-0 
> Testing ACK [7e37329](https://github.com/bitcoin-core/gui/commit/7e373294a5ae819099c39d9d03d1f5a311d63cfc) on Windows Windows 10.0.18363 Build 18363 and Ubuntu 20.04.
> 
> Looks good to me!

Currently, the top commit is 401a5e70de42e6ddf0b5bc79d96ea8d670851860 rather 7e37329 :)

**Comment by jarolrod** [comment]
Added a colon to the `Third party transaction URLs` feature under the `Display` Tab as suggested by @hebasto. Check 88ecffa.

File: src/qt/forms/optionsdialog.ui [hunk 1]
```diff
@@ -33,7 +33,7 @@
           <string>Automatically start %1 after logging in to the system.</string>
          </property>
          <property name="text">
-          <string>&amp;Start %1 on system login</string>
```
Comment (luke-jr) [inline src/qt/forms/optionsdialog.ui:36]: Not sure it's a good idea to tie the shortcut to PACKAGE_NAME... This option is more about startup anyway: [S] is a good fit.

File: src/qt/forms/optionsdialog.ui [hunk 2]
```diff
@@ -58,7 +58,7 @@
            <string>Disables some advanced features but all blocks will still be fully validated. Reverting this setting requires re-downloading the entire blockchain. Actual disk usage may be somewhat higher.</string>
           </property>
           <property name="text">
-           <string>Prune &amp;block storage to</string>
+           <string>&amp;Prune block storage to:</string>
```
Comment (luke-jr) [inline src/qt/forms/optionsdialog.ui:61]: ACK despite no need

File: src/qt/forms/optionsdialog.ui [hunk 3]
```diff
@@ -105,7 +105,7 @@
          <item>
           <widget class="QLabel" name="databaseCacheLabel">
            <property name="text">
-            <string>Size of &amp;database cache</string>
+            <string>&amp;Size of database cache:</string>
```
Comment (luke-jr) [inline src/qt/forms/optionsdialog.ui:108]: Don't see a good option here. Maybe just fall back to [I]?

File: src/qt/forms/optionsdialog.ui [hunk 4]
```diff
@@ -212,7 +212,7 @@
              <string>Whether to show coin control features or not.</string>
             </property>
             <property name="text">
-             <string>Enable coin &amp;control features</string>
+             <string>&amp;Enable coin control features</string>
```
Comment (luke-jr) [inline src/qt/forms/optionsdialog.ui:215]: Meh, okay.

File: src/qt/forms/optionsdialog.ui [hunk 5]
```diff
@@ -275,7 +275,7 @@
           <string>Connect to the Bitcoin network through a SOCKS5 proxy.</string>
          </property>
          <property name="text">
-          <string>&amp;Connect through SOCKS5 proxy (default proxy):</string>
+          <string>Connect through SOCKS&amp;5 proxy (default proxy):</string>
```
Comment (luke-jr) [inline src/qt/forms/optionsdialog.ui:278]: ACK

File: src/qt/forms/optionsdialog.ui [hunk 6]
```diff
@@ -583,7 +583,7 @@
           <string>Show only a tray icon after minimizing the window.</string>
          </property>
          <property name="text">
-          <string>&amp;Minimize to the tray instead of the taskbar</string>
+          <string>Minimize to the &amp;tray instead of the taskbar</string>
```
Comment (luke-jr) [inline src/qt/forms/optionsdialog.ui:586]: ACK

**Review by luke-jr — CHANGES_REQUESTED** [review]

**Comment by hebasto** [comment]
https://www.nngroup.com/articles/ui-copy/ (the "Guidelines for Command Shortcuts" part) could be relevant.

**Comment by promag** [comment]
Concept ACK.

**Comment by jonasschnelli** [comment]
Have we though how this would behave after translations? I guess the translators would define the shortcuts for non English languages?

**Comment by hebasto** [comment]
Here are some excerpts from Qt docs:
- https://doc.qt.io/qt-5/linguist-overview.html
> Keyboard accelerators may need to be changed but without introducing conflicts... We cannot use a letter that is already in use - unless we change several accelerators.

- https://doc.qt.io/qt-5/linguist-translators.html#changing-keyboard-accelerators
> Conflicts with other Alt key accelerators must be avoided within a context. Note that some Alt key accelerators, usually those on the menu bar, may apply in other contexts.

@jonasschnelli 
> Have we though how this would behave after translations? I guess the translators would define the shortcuts for non English languages?

With the current translation process it is a translators' responsibility to (re)define shortcuts, right?

**Comment by jarolrod** [comment]
leaving up for grabs

**Pull request closed without merging**